build_with_cmake/node-addon-api/hello.cc

```diff
@@ -0,0 +1,14 @@
+#include <napi.h>
+
+Napi::String Method(const Napi::CallbackInfo& info) {
+  Napi::Env env = info.Env();
+  return Napi::String::New(env, "Hello, world!");
+}
+
+Napi::Object Init(Napi::Env env, Napi::Object exports) {
```

Comment:
Ditto.

Reply:
Fixed in https://github.com/nodejs/node-addon-examples/pull/114.